src/js/components/Anchor.js

```diff
@@ -17,11 +17,13 @@ export default class Anchor extends Component {
   }
 
   _onClick (event) {
-    const { method, onClick, path} = this.props;
+    const { method, onClick, path, disabled } = this.props;
     const { router } = this.context;
 
     event.preventDefault();
 
+    if (disabled) return;
```

Comment:
Thanks for your contribution. Would you mind to use the following pattern instead?

```
if (!disabled) {
  ...
}
```

This is very minor, and it is just for consistency's sake.